sqlite-android/src/main/java/io/requery/android/database/sqlite/SQLiteDatabase.java

```diff
@@ -826,8 +826,12 @@ private static void ensureFile(String path) {
         File file = new File(path);
         if (!file.exists()) {
             try {
-                if (!file.getParentFile().mkdirs()) {
-                    Log.e(TAG, "Couldn't mkdirs " + file);
+                if (!file.getParentFile().exists()) {
```

Comment:
Maybe combine into one?
```suggestion
                if (!file.getParentFile().exists() && !file.getParentFile().mkdirs()) {
```